ksqldb-rest-model/src/test/java/io/confluent/ksql/rest/entity/KafkaStreamsStateCountTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import io.confluent.ksql.util.KsqlException;
+import org.apache.kafka.streams.KafkaStreams;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Map;
+
+@SuppressWarnings("SameParameterValue")
+public class KafkaStreamsStateCountTest {
```

Comment:
With the changes I suggested to make this object serialize as a JSON object you can add tests such as:

```java
@Test
  public void shouldRoundTripWhenEmpty() {
    // When:
    final String json = assertDeserializedToSame(kafkaStreamsStateCount);

    // Then:
    assertThat(json, is("{}"));
  }

  @Test
  public void shouldRoundTripWhenNotEmpty() {
    // Given:
    kafkaStreamsStateCount.updateStateCount(State.RUNNING, 2);
    kafkaStreamsStateCount.updateStateCount(State.CREATED, 10);

    // When:
    final String json = assertDeserializedToSame(kafkaStreamsStateCount);

    // Then:
    assertThat(json, is("{"
        + "\"CREATED\":10,"
        + "\"RUNNING\":2"
        + "}"));
  }

  private static String assertDeserializedToSame(final KafkaStreamsStateCount original) {
    try {
      final String json = MAPPER.writeValueAsString(original);

      final KafkaStreamsStateCount deserialized = MAPPER
          .readValue(json, KafkaStreamsStateCount.class);

      assertThat(deserialized, is(original));

      return json;
    } catch (Exception e) {
      throw new AssertionError("Failed to round trip", e);
    }
  }
```

Reply:
added the tests